rules/windows/defense_evasion_masquerading_werfault.toml

```diff
@@ -6,27 +6,34 @@ updated_date = "2020/11/03"
 
 [rule]
 author = ["Elastic"]
-description = "Identifies a suspicious WerFault command line parameter, which may indicate an attempt to run unnoticed."
+description = """
+Identifies suspicious instances of the Windows Error Reporting process (WerFault.exe or Wermgr.exe) with a command-line value equal to the process executable and
+performing outgoing network connections. This may indicate a masquerading attempt to evade suspicious child process
+related behavior detections.
```

Comment:
```suggestion
Identifies suspicious instances of the Windows Error Reporting process (WerFault.exe or Wermgr.exe) with matching command-line and process executable values performing outgoing network connections. This may be indicative of a masquerading attempt to evade suspicious child process behavior detections.
```

Reply:
will need to re-lint